schema-engine/sql-migration-tests/tests/migrations/postgres/multi_schema.rs

```diff
@@ -1392,6 +1398,7 @@ fn migration_with_shadow_database(api: TestApi) {
 
     let namespaces = Namespaces::from_vec(&mut vec![String::from("one"), String::from("two")]);
 
+    api.raw_cmd("DROP DATABASE IF EXISTS shadow");
```

Comment:
This test was otherwise not able to run twice against the same db server. Don't know why this wasn't a problem so far but it already behaved that way on master. 🤷